clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -506,6 +511,20 @@
     private static final long NO_CURRENT_THREAD = -1L;
     private static final AtomicInteger CONSUMER_CLIENT_ID_SEQUENCE = new AtomicInteger(1);
     private static final String JMX_PREFIX = "kafka.consumer";
+    /**
+     * APIs used by KafkaConsumer
+     */
+    private static final List<Short> USED_API_KEYS = Arrays.asList(
```

Comment:
Also, maybe we could call this `CONSUMER_APIS` and get rid of the comment, which doesn't add much value.